arangodb-net-standard/DocumentApi/Models/HeadDocumentHeader.cs

```diff
@@ -8,17 +8,29 @@ public class HeadDocumentHeader
 
         public string IfNoneMatch { get; set; }
 
+        /// <summary>
+        /// Gets or sets the Id of a stream transaction.
+        /// </summary>
+        public string TransactionId { get; set; }
+
         public WebHeaderCollection ToWebHeaderCollection()
         {
             WebHeaderCollection collection = new WebHeaderCollection();
             if (IfMatch != null)
             {
                 collection.Add(HttpRequestHeader.IfMatch, $"\"{IfMatch}\"");
             }
+
             if (IfNoneMatch != null)
             {
                 collection.Add(HttpRequestHeader.IfNoneMatch, $"\"{IfNoneMatch}\"");
             }
+
+            if (TransactionId != null)
+            {
+                collection.Add("x-arango-trx-id", TransactionId);
```

Comment:
Can we make a constant for `x-arango-trx-id`?   This string literal also appears in CursorApiClient.cs and CursorApiClientTest.cs

Reply:
Like a global constant(which folder to put) or private constant within _HeadDocumentHeader_ and _CursorApiClient_?

Reply:
The motivation for the comment is DRY principle - so declaring in a shared place so it can be reused (not a private constant then).  A suitable place might be a new static class named "CustomHttpHeaders", for example.  I'd welcome any other sensible suggestion.

Reply:
I would suggest `ArangoHttpHeaders` but `CustomHttpHeaders` works equally well, as it's mainly used inside the library.